src/api/api.go

```diff
@@ -21,7 +21,7 @@ func (a *Api) init(debug bool) {
 	gin.DisableConsoleColor()
 	a.engine = gin.Default()
 	// Healthcheck
-	a.engine.GET("/ping", handlePing)
+	a.engine.GET("/v1/ping", handlePing)
```

Comment:
This breaks compatibility with the Metronome API. The `/ping` endpoint is separate from the `/v1/*` endpoints.

https://dcos.github.io/metronome/docs/generated/api.html